src/assets/stylesheets/base/_font.scss

```diff
@@ -0,0 +1,12 @@
+@font-face {
```

Comment:
You still need another `@font-face` to load the bold font 😢 
I meant you can declare 2 `@font-face`, 1 for normal, 1 for bold, and you can set them to have the same name. Then you can use it in the body tag like below.